types/config.ts

```diff
@@ -17,7 +17,6 @@ export type NavigationPluginConfig = {
   gql: PluginConfigGraphQL;
   i18nEnabled: boolean;
   pruneObsoleteI18nNavigations: boolean;
-  slugify: Record<string, unknown>;
```

Comment:
I just realized that this is not mentioned in the readme. Also, this commit might be considered a breaking change since some users might have their slugify already configured.